shell/platform/android/android_shell_holder.cc

```diff
@@ -32,6 +32,47 @@
 
 namespace flutter {
 
+/// Inheriting ThreadConfigurer and use Android platform thread API to configure
+/// the thread priorities
+class PlatformAndroidThreadConfig : public fml::Thread::ThreadConfig {
+ public:
+  using fml::Thread::ThreadConfig::ThreadConfig;
+
+  void SetCurrentThreadPriority() override {
```

Comment:
In here, I think may be not all user want to set the thread priority, so I have provided an empty implementation for the user who don't want to set priority. Because of he don't want to set the priority, he no necessary to override the method.